src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/GridViewWithSelectionViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+#if XAMARIN
+using Windows.UI.Xaml.Controls;
+#else
+using Windows.UI.Xaml.Controls;
+#endif
+
+namespace Uno.UI.Samples.Presentation.SamplePages.GridView
+{
+	public class GridViewWithSelectionViewModel : ViewModelBase
+	{
+
+		public GridViewWithSelectionViewModel(CoreDispatcher coreDispatcher) : base(coreDispatcher)
+		{
+			// TODO: restore commands
+			//Build(b => b
+			//	.Properties(pb => pb
+			//		.Attach("SampleItems", () => GetSampleItems())
+			//		.Attach(SelectionMode, () => ListViewSelectionMode.Single)
+			//		.AttachUserCommand("NoneSelection", async ct => SelectionMode.Value.OnNext(ListViewSelectionMode.None))
+			//		.AttachUserCommand("SingleSelection", async ct => SelectionMode.Value.OnNext(ListViewSelectionMode.Single))
+			//		.AttachUserCommand("MultipleSelection", async ct => SelectionMode.Value.OnNext(ListViewSelectionMode.Multiple))
+			//	)
+			//);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5930374)